platform/darwin/src/MGLLoggingConfiguration.mm

```diff
@@ -18,6 +50,14 @@ + (instancetype)sharedConfiguration
     return sharedConfiguration;
 }
 
+- (id)init{
+    if(self = [super init]){
+        _coreLoggingObserver = std::make_unique<mbgl::MGLCoreLoggingObserver>();
```

Comment:
Does this need to be an ivar, if we're just passing it to `setObserver`?

Reply:
Good point. Already fixed it.